ui/imports/shared/popups/send/controls/TokenBalancePerChainDelegate.qml

```diff
@@ -48,7 +48,12 @@ StatusListItem {
         let balance = !!model && !!model.currentCurrencyBalance ? model.currentCurrencyBalance : 0
         return root.formatCurrentCurrencyAmount(balance)
     }
-    asset.name: symbol ? Style.png("tokens/" + symbol) : ""
+    // Community assets have a dedicated image streamed from status-go
```

Comment:
both above and below does the same thing. Can we create a utils function for this instead of repeating?

Reply:
`Constants.tokenIcon` pls :)

Reply:
Done, thanks for the heads up